CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ have notable changes.
 
 Changes since v2.26
 
+### Additions
+- Bundling resources to application can now be enabled or disabled in user interface with configuration option `:enable-cart`, which defaults to `true`. (#2720)
```

Comment:
Perhaps we should write directly that "Shopping cart can now be ... " as it says directly what it does.